test/SaveHandler/MongoDBTest.php

```diff
@@ -43,18 +43,25 @@ class MongoDBTest extends \PHPUnit_Framework_TestCase
     public function setUp()
     {
         if (!extension_loaded('mongo')) {
-            $this->markTestSkipped('Zend\Session\SaveHandler\MongoDB tests are not enabled due to missing Mongo extension');
+            $this->markTestSkipped(
+                'Zend\Session\SaveHandler\MongoDB tests are not enabled due to missing Mongo extension'
+            );
         }
 
-        $this->options = new MongoDBOptions([
-            'database' => 'zf2_tests',
-            'collection' => 'sessions',
-        ]);
+        $this->options = new MongoDBOptions(
+            [
+                'database' => 'zf2_tests',
+                'collection' => 'sessions',
+            ]
+        );
```

Comment:
no. The only wrong thing is to change unrelated lines in the PR. Both indentations are correct.